python/logcat-color_select/files/none

```diff
@@ -0,0 +1 @@
+-
```

Comment:
What is this for?

Reply:
Actullay I don't quite understand how the select port group works. This is copied from https://github.com/macports/macports-ports/blob/master/python/flake8_select/files/none

Reply:
@mojca do you know how this works?

Reply:
Seems `-` instructs MacPorts to remove the target symbolic link. In this case the link /opt/local/bin/logcat-color is removed when ```port select --set logcat-color none``` is invoked

https://github.com/macports/macports-base/blob/8dda587a7359d8aa71fea3ad9744ba7019dcd1f2/src/macports1.0/macports.tcl#L4371-L4376

Reply:
Yes, this is needed in order to delete the symlink. I slightly wonder if this is not a slight overkill, but well ... it's up to maintainer.